### DISCUSSION TITLE
Hosting in a kubernetes cluster?

### DISCUSSION THREAD

**Discussion by achainarong** [supabase/supabase#3073] [Questions] [answered]
Hey guys i just saw that it is possible to self host supa base with a docker compose stack.
Is it also possible to migrate it to a kubernetes cluster.

**Comment by darora** [comment]
Hey @achainarong - there's no reason Supabase wouldn't function when hosted in a k8s cluster.

We don't currently maintain a Helm chart or similar, though. I'd be happy to review the PR if you want to put something up!

**Reply by achainarong** [reply]
What would the acceptance criteria for the pullrequest be? I never worked on an opensource project before so im not familiar with the steps necessary to be able to contribute to the project.

**Comment by J0** [comment, marked as answer]
Hey @achainarong,

It's been a while -- hope you're still doing fine! There's now a community supported set of [open source helm 3 charts](https://github.com/supabase-community/supabase-kubernetes) Feel free to file a pull request there. The acceptance criteria would be determined by the maintainer so I suggest you lease with him on the repo.

Thanks!
Joel

**Comment by rossigee** [comment]
It could be a waste of time trying to use those charts or raise PRs against them just now. They [don't appear to be have been maintained](https://github.com/orgs/supabase/discussions/36048) for quite a while now.
